app/src/main/res/layout/activity_create_profile.xml

```diff
@@ -9,21 +10,14 @@
         android:layout_height="wrap_content"
         android:orientation="vertical">
 
-        <android.support.v7.widget.Toolbar
-            android:id="@+id/toolbar3"
+        <ImageView
+            android:id="@+id/large_logo"
```

Comment:
If the id isn't used you don't need to specify it, it only clutters R.java, but it's not really a problem either